tests/Blocks/BoardListBlock.test.tsx

```diff
@@ -0,0 +1,99 @@
+import "@testing-library/jest-dom/extend-expect";
+
+import * as boardListBlockStories from "stories/Blocks/BoardListBlock.stories";
+
+import { render, screen } from "@testing-library/react";
+import React from "react";
+
+import { action } from "@storybook/addon-actions";
+import { userEvent } from "@storybook/testing-library";
+import { composeStories } from "@storybook/testing-react";
+
+jest.mock("@storybook/addon-actions");
+
+const { Simple } = composeStories(boardListBlockStories);
+
+// 1) clicking the board icon takes you to the board
+
+describe("When a board icon is clicked", () => {
+  render(<Simple />);
+  const boardToClick = Simple.args!.boards![2];
+  const boardIcon = screen.getByRole("link", {
+    name: `${boardToClick.slug} icon`,
+  });
+
+  it("takes you to the board", async () => {
+    await userEvent.click(boardIcon);
+    expect(boardIcon).toHaveAttribute("href", "#slug");
```

Comment:
Shouldn't this be `boardToClick.slug` rather than `"#slug"`?

Reply:
Gonna be honest, I'm not sure either 😂 I grabbed it from other places like https://github.com/search?q=repo%3ABobaBoard%2Fboba-components%20toHaveAttribute(%22href%22%2C%20%22%23slug%22)%3B&type=code

I think it's because of how it's set up that the `link` property is `{ href: "#slug", onClick: action("#slug") },` and unless my javascript life is a lie, I don't think you can access another property (like `slug`) from there?

Reply:
I solved my own question: it's because all the links we defined in boards use `#slug` for their url instead of their own board name (like `#gore`). This means we cannot really distinguish among them.

It's fine for now though, all looks good to me!